test/integration/068_partial_parsing_tests/test_partial_parsing.py

```diff
@@ -477,6 +477,8 @@ def test_postgres_skip_macros(self):
         # initial run so we have a msgpack file
         self.setup_directories()
         self.copy_file('test-files/model_one.sql', 'models/model_one.sql')
+        # use empty_schema file for bug #4850
+        self.copy_file('test-files/empty_schema.yml', 'models/eschema.yml')
```

Comment:
This test used to run always without this file being present, but now runs always with this file being. So we effectively lost the integration test scenario where this file doesn't exist at all. Do we care about that? We may not; I don't know enough about this functionality.

Reply:
No, we don't care about that.